src/guide/lnurl_pay.md

```diff
@@ -13,11 +13,15 @@
 let lnurl_pay_url = "lightning@address.com";
 
 if let Ok(LnUrlPay{data: pd}) = parse(lnurl_pay_url).await {
-    // TODO Show payment details in UI, read user input
     let amount_msat = pd.min_sendable;
     let comment = "Test payment".to_string();
     
-    sdk.lnurl_pay(amount_msat, Some(comment), pd).await?;
+    sdk.lnurl_pay(LnUrlPayRequest { 
+        req_data: pd, 
```

Comment:
Might want to rename all `req_data` params as `data` and merge this when that change is applied on breez-sdk.

Reply:
Will do. Are you doing the renaming in your PR?